sdk/python/jobs/automl-standalone-jobs/automl-forecasting-github-dau/auto-ml-forecasting-github-dau.ipynb

```diff
@@ -0,0 +1,841 @@
+    "\n",
+    "To connect to a workspace, we need identifier parameters - a subscription, resource group and workspace name. We will use these details in the `MLClient` from `azure.ai.ml` to get a handle to the required Azure Machine Learning workspace. We use the default [default azure authentication](https://docs.microsoft.com/en-us/python/api/azure-identity/azure.identity.defaultazurecredential?view=azure-python) for this tutorial. Check the [configuration notebook](../../configuration.ipynb) for more details on how to configure credentials and connect to a workspace."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "credential = DefaultAzureCredential()\n",
+    "ml_client = None\n",
+    "try:\n",
+    "    ml_client = MLClient.from_config(credential)\n",
+    "except Exception as ex:\n",
+    "    print(ex)\n",
+    "    # Enter details of your AML workspace\n",
+    "    subscription_id = \"<SUBSCRIPTION_ID>\"\n",
+    "    resource_group = \"<RESOURCE_GROUP>\"\n",
+    "    workspace = \"<AML_WORKSPACE_NAME>\"\n",
+    "    ml_client = MLClient(credential, subscription_id, resource_group, workspace)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "hideCode": false,
+    "hidePrompt": false
+   },
+   "source": [
+    "### Show Azure ML Workspace information"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "hideCode": false,
+    "hidePrompt": false
+   },
+   "outputs": [],
+   "source": [
+    "workspace = ml_client.workspaces.get(name=ml_client.workspace_name)\n",
+    "\n",
+    "output = {}\n",
+    "output[\"Workspace\"] = ml_client.workspace_name\n",
+    "output[\"Subscription ID\"] = ml_client.connections._subscription_id\n",
+    "output[\"Resource Group\"] = workspace.resource_group\n",
+    "output[\"Location\"] = workspace.location\n",
+    "output"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "hideCode": false,
+    "hidePrompt": false
+   },
+   "source": [
+    "# 2. Data\n",
+    "\n",
+    "We will use github active user (DAU) count for model training. The data is stored in a tabular format.\n",
+    "\n",
+    "With Azure Machine Learning MLTables you can keep a single copy of data in your storage, easily access data during model training, share data and collaborate with other users. \n",
+    "Below, we will upload the data by creating an MLTable to be used for training.\n",
+    "\n",
+    "**NOTE:** In this PRIVATE PREVIEW we're defining the MLTable in a separate folder and .YAML file.\n",
+    "In later versions, you'll be able to do it all in Python APIs."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {
+    "hideCode": false,
+    "hidePrompt": false
+   },
+   "outputs": [],
+   "source": [
+    "from helpers.generate_ml_table import create_ml_table\n",
```

Comment:
nit: should we have all imports at top?

Reply:
I referred to some notebooks and followed the same pattern. v1 notebooks and some v2 notebooks as well. e.g. https://github.com/Azure/azureml-examples/blob/a6154d6c022eb07c8b506c7523e5d76680df9915/sdk/python/jobs/pipelines/1h_automl_in_pipeline/automl-forecasting-in-pipeline/automl-forecasting-in-pipeline.ipynb.
Please let me know if this change is required. Thanks!